### PR TITLE
Formatter should not try to format non-Rust files

### LINKED ISSUE #106261: `x fmt` tries to format non-rust files
```
formatting modified file compiler/rustc_error_messages/locales/en-US/borrowck.ftl
error: unknown start of token: `
 --> /home/nilsh/projects/rust/compiler/rustc_error_messages/locales/en-US/borrowck.ftl:2:33
  |
2 |     cannot move a value of type `{$ty}`
  |                                 ^
  |
help: Unicode character '`' (Grave Accent) looks like ''' (Single Quote), but it is not
  |
2 |     cannot move a value of type '{$ty}`
  |                                 ~

error: unknown start of token: `
```

When there was a change in a `.ftl` or other non-rust file, bootstrap tries to format it, which makes rustfmt very sad.

cc @albertlarsan68 and @jyn514 who broke this in #105702

### PR DESCRIPTION
**Pull request by chenyukang**
Fixes #106261

### CODE REVIEW

File: src/bootstrap/format.rs [hunk 1]
```diff
@@ -78,7 +78,7 @@ fn update_rustfmt_version(build: &Builder<'_>) {
 /// rust-lang/master and what is now on the disk.
 ///
 /// Returns `None` if all files should be formatted.
-fn get_modified_files(build: &Builder<'_>) -> Option<Vec<String>> {
+fn get_modified_fs_files(build: &Builder<'_>) -> Option<Vec<String>> {
```
Comment (albertlarsan68) [inline src/bootstrap/format.rs:81]:
```suggestion
fn get_modified_rs_files(build: &Builder<'_>) -> Option<Vec<String>> {
```
Typo ?

**Review by jyn514 — APPROVED** [review]
Thanks! r=me with @albertlarsan68's comment fixed.

File: src/bootstrap/format.rs (hunk 1 above)
Comment (chenyukang) [reply src/bootstrap/format.rs]: yes, fixed!

**Pull request merged**